ChakraCore Samples/Hello World/Linux_OSX_Shared/README.md

```diff
@@ -0,0 +1,25 @@
+# ChakraCore Hello-world Sample
+This sample shows how to embed ChakraCore with [JavaScript Runtime (JSRT) APIs](http://aka.ms/corejsrtref) on Linux/OS X, and complete the following tasks:
```

Comment:
Add "as shared library" for clarification.

This sample shows how to embed ChakraCore as shared library with ...